app/views/registration/new.html.erb

```diff
@@ -4,7 +4,7 @@
             <div class="uk-panel uk-panel-box rounded-border">
                 <h2>Regisztráció</h2>
 
-                <%= form_tag('/auth/register') do %>
+                <%= form_tag('/register/create') do %>
```

Comment:
Megint csak erdemes valami url helpert hasznalni. Lehet a generaltat is.

Peldaul

```ruby
form_tag(registrations_path) do
```

Vagy

```ruby
form_tag url_for(controller: :registrations, action: :create) do
```

Az elso az inkabb koveti a Railses konvenciot.